src/platform/item/linked-wearable-mappings.ts

```diff
@@ -173,3 +173,103 @@ export namespace Mapping {
     RangeMapping._fromLessThanOrEqualTo
   ])
 }
+
+export type Network = 'mainnet' | 'matic' | 'sepolia' | 'amoy'
```

Comment:
What do you think about renaming the `Network` type to `ContractNetwork`? It would be helpful to differentiate between other variables exported here that are named like it.
```suggestion
export type ContractNetwork = 'mainnet' | 'matic' | 'sepolia' | 'amoy'
```